src/test/__mocks__/pickLists.ts

```diff
@@ -0,0 +1,379 @@
+import { PickListOption } from '@/types/gqlTypes';
+
+export const pickListData: Record<string, PickListOption[]> = {
```

Comment:
You should be able to pull these from the gqlEnums file

Reply:
I actually don't think that I can. I need the `groupLabel` and `groupCode`, which I don't think are included in that file. But I'll probably end up removing this since I don't think I'll end up leaving it in storybook beyond the dev on this feature.